UI/window-basic-stats.hpp

```diff
@@ -29,6 +30,10 @@ class OBSBasicStats : public QWidget {
 
 	QTimer timer;
 
+	uint64_t totalSeconds = 0;
+
+	std::vector<long double> bitrates;
```

Comment:
Might be worth calling reserve() on this since we have a good idea how many samples it will have.

Reply:
It is now set to reserve(15).